packages/api/webpack.config.js

```diff
@@ -44,13 +47,33 @@ module.exports = {
   ],
   output: {
     libraryTarget: 'commonjs2',
-    filename: '[name]/index.js'
+    filename: '[name]/index.js',
+    devtoolModuleFilenameTemplate: (info) => {
```

Comment:
The only point of this is to avoid having absolute paths to code on our machines show up in Cloudwatch logs error traces when we are deploying our tasks with symlinked packages. It ensures that a relative path from the module is used instead of the absolute path to the symlinked code.